content/go/concepts/strings/terms/map/map.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: 'Map()'
+Description: 'Returns a copy of the provided string with its characters converted according to the mapping function.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`strings.Map()`** method changes all of the characters of a given string according to the changes specified in the given mapping function.
+
+## Syntax
+
+```pseudo
+Map(mappingFunct(r),  s)
+```
+
+The `strings.Map()` function takes two parameters:
+
+- `mappingFunct(r)`: A function and a character (type rune) that the original character will be replaced with.
```

Comment:
We don't have an entry on runes so it would be helpful here to making a differentiation here

Reply:
edit: I'm going to link the datatypes entry here

Reply:
Thank you for adding the link!